rfc8446.xml

```diff
@@ -4444,138 +5027,158 @@ might change, the ServerHello format might change, and the TLSCiphertext format
-  <t>IANA [SHALL update/has updated] this registry to include a “TLS
-1.3” column which lists the messages in which the extension may
-appear. This column [SHALL be/has been]
-initially populated from the table in <xref target="extensions"/>
-with any extension not listed there marked as “-“ to indicate that
-it is not used by TLS 1.3.</t>
+via Specification Required <xref target="RFC8126"></xref>.  Values with the
+first byte 255 (decimal) are reserved for Private Use <xref
+target="RFC8126"></xref>.</t>
+  <t>IANA has updated this registry to include the
+"key_share", "pre_shared_key", "psk_key_exchange_modes",
+"early_data", "cookie", "supported_versions",
+"certificate_authorities", "oid_filters", "post_handshake_auth", and
+"signature_algorithms_cert" extensions with the values defined in this
+document and the "Recommended" value of "Yes".</t>
+  <t>IANA has updated this registry to include a
+"TLS&nbsp;1.3" column which lists the messages in which the extension may
+appear. This column has been initially populated from the table in <xref
+target="extensions"/>, with any extension not listed there marked as "-" to
+indicate that it is not used by TLS 1.3.</t>
 </list></t>
 
-<t>In addition, this document defines two new registries to be maintained
+<t>In addition, this document defines two new registries that are maintained
 by IANA:</t>
-
 <t><list style="symbols">
-  <t>TLS SignatureScheme Registry: Values with the first byte in the range
-0-253 (decimal) are assigned via Specification Required <xref target="RFC8126"/>.
-Values with the first byte 254 or 255 (decimal) are reserved for Private
-Use <xref target="RFC8126"/>. Values with the first byte in the range 0-6 or with the
-second byte in the range 0-3 that are not currently allocated are reserved for
-backwards compatibility.
-This registry SHALL have a “Recommended” column.
-The registry [shall be/ has been] initially populated with the values described in
-<xref target="signature-algorithms"/>. The following values SHALL be marked as
-“Recommended”: ecdsa_secp256r1_sha256, ecdsa_secp384r1_sha384,
+  <t>"TLS SignatureScheme" registry: Values with the first byte in the range
+0-253 (decimal) are assigned via Specification Required <xref
+target="RFC8126"/>. Values with the first byte 254 or 255 (decimal) are
+reserved for Private Use <xref target="RFC8126"/>. Values with the first byte
+in the range 0-6 or with the second byte in the range 0-3 that are not
+currently allocated are reserved for backward compatibility.
+This registry has a "Recommended" column.
+The registry has been initially populated with the values described in
+<xref target="signature-algorithms"/>. The following values are marked as
+"Recommended": ecdsa_secp256r1_sha256, ecdsa_secp384r1_sha384,
 rsa_pss_rsae_sha256, rsa_pss_rsae_sha384, rsa_pss_rsae_sha512,
 rsa_pss_pss_sha256, rsa_pss_pss_sha384, rsa_pss_pss_sha512, and ed25519.</t>
-  <t>TLS PskKeyExchangeMode Registry: Values in the
+
```

Comment:
Because we're creating the TLS Signature Scheme registry, we need to say how future values are assigned.  I'd like to propose:

    Future extensions MUST define the value of the Recommended column.

Reply:
Addressed in #6